rn/Teacher/src/modules/inbox/Compose.js

```diff
@@ -285,7 +284,7 @@ export class Compose extends PureComponent<ComposeProps & OwnProps, ComposeState
             contentContainerStyle={{ flexGrow: 1, paddingBottom: 16 }}
           >
             { Boolean(this.props.showCourseSelect) &&
-              <TouchableHighlight testID='compose.course-select' underlayColor='#fff' style={styles.wrapper} onPress={this.props.canSelectCourse ? this.selectCourse : undefined}>
+              <TouchableHighlight testID='compose.course-select' underlayColor='#ffffff00' style={styles.wrapper} onPress={this.props.canSelectCourse ? this.selectCourse : undefined}>
```

Comment:
Is this essentially clear color?

Reply:
yes, it's a clear white. I'm not sure why it matters, but we do it in a bunch of places. I'm pretty sure white is wrong.